.env.example

```diff
@@ -19,3 +19,6 @@ LD_FRONTEND_KEY=
 # LD_RELAY_STREAM_URL=
 # Leave LD_API_KEY blank unless you need to connect to the API
 # LD_API_KEY=
+
+# Uncomment the following section to send metrics to Datadog. This is wired to the Support Service (Dev) account.
+# DD_API_KEY=83821f1a517bc8eb9a6573e7fede9568
```

Comment:
@rmanalan we need to rotate this key, we can add secrets into the circleci config, this is a public repo

Reply:
Ugh... sorry about that. I didn't realize this was public. Will rotate.